### ISSUE TITLE
Add evolvotron_complexity script

### ISSUE THREAD

**Issue by denilsonsa** [WickedSmoke/evolvotron#6]
The [old Evolvotron website](https://www.timday.com/share/evolvotron/) mentions:

> Alpt of [FreakNet](http://freaknet.org/) has some [nice results](https://web.archive.org/web/20121020024013/http://freaknet.org/alpt/misc/fatrial/evcomplexity/) (the .jpgs) from a [script][] which evolves evolvotron images automatically by comparing their complexity.

Those URLs are long gone, but thankfully [the script is still archived in the WayBack Machine][script].

My suggestion is to add that script to the Evolvotron project, so that it won't be lost.

[script]: https://web.archive.org/web/20121020024027/http://freaknet.org/alpt/src/misc/evolvotron_complexity

**Comment by WickedSmoke** [comment]
That script is outdated and causes evolvotron_render to crash.

```
terminate called after throwing an instance of 'boost::wrapexcept<boost::program_options::too_many_positional_options_error>'
  what():  too many positional options have been specified on the command line
zsh: done                           cat good_1.xml | 
zsh: IOT instruction (core dumped)  ./evolvotron_render -s 1024 1024 rendered_image.ppm
```

Boost is such garbage.  Anyways here's a patch for it:

```
87,91c87,91
< #resolution="4096 4096"
< #resolution="2048 2048"
< resolution="1024 1024"
< #resolution="512 512"
< #resolution="256 256"
---
> #resolution="4096x4096"
> #resolution="2048x2048"
> resolution="1024x1024"
> #resolution="512x512"
> #resolution="256x256"
110,113d109
< render() {
<       evolvotron_render -s $resolution $@
< }
< 
116c112
<       cat $1 | render $2; bzip2 -f $2
---
>       cat $1 | evolvotron_render -s $resolution -o $2; bzip2 -f $2
```

Now with that done there's still the issue of copyright.  AlpT did not specify any license so someone would have to contact him to get permission to distribute it.

**Comment by denilsonsa** [comment]
> Now with that done there's still the issue of copyright. AlpT did not specify any license so someone would have to contact him to get permission to distribute it.

Well, the [script][s] was found at /[alpt][]/[src][]/[misc][]/[evolvotron_complexity][s]. Thankfully, inside the [src][] directory we can find:

> In this place you'll find some code of mine.
> All the code is released under GPL-2.
> Please see the COPYING file for more info.

And the COPYING file, although not archived, looks like the beginning of the GPL-2 license.

---

I'll admit I found the script and it sounded interesting, but I hadn't tried it myself. (In fact, it was already too late into the night.)

Thanks for taking a look at it. I hope I could clarify the license.

[s]: https://web.archive.org/web/20121020024027/http://freaknet.org/alpt/src/misc/evolvotron_complexity
[misc]: https://web.archive.org/web/20121020024027/http://freaknet.org/alpt/src/misc/
[src]: https://web.archive.org/web/20080827160229/http://www.freaknet.org/alpt/src/
[alpt]: https://web.archive.org/web/20080705013042/http://freaknet.org/alpt/

**Comment by WickedSmoke** [comment]
Ok, the script has been fixed and the license noted in commit 673aae37.
